proto/ibc/applications/fee/v1/query.proto

```diff
@@ -48,6 +49,16 @@ service Query {
     option (google.api.http).get = "/ibc/apps/fee/v1/total_timeout_fees/port/{packet_id.port_id}/channel/"
                                    "{packet_id.channel_id}/sequence/{packet_id.sequence}";
   }
+
+  // FeeEnabledChannels returns a list of all fee enabled channels
+  rpc FeeEnabledChannels(QueryFeeEnabledChannelsRequest) returns (QueryFeeEnabledChannelsResponse) {
+    option (google.api.http).get = "/ibc/apps/fee/v1/fee_enabled";
+  }
+
+  // FeeEnabledChannel returns true if the provided port and channel identifiers belong to a fee enabled channel
+  rpc FeeEnabledChannel(QueryFeeEnabledChannelRequest) returns (QueryFeeEnabledChannelResponse) {
+    option (google.api.http).get = "/ibc/apps/fee/v1/fee_enabled/{port_id}/{channel_id}";
```

Comment:
Should we do `/ibc/apps/fee/v1/fee_enabled/port/{port_id}/channel/{channel_id}` like in other queries?